saleor/graphql/schema.graphql

```diff
@@ -143,8 +143,8 @@ type AssignNavigation {
 
 type Attribute implements Node {
   id: ID!
-  productTypes(before: String, after: String, first: Int, last: Int): ProductTypeCountableConnection
```

Comment:
https://github.com/graphql-python/graphene-django/releases/tag/v2.5.0

It was changed to required, will it be an issue?

Reply:
No it should be fine. (https://github.com/mirumee/saleor/pull/4653)